prac_05/word_occurrences.py

```diff
@@ -0,0 +1,12 @@
+word_count = {}
+sentence = input("Enter words: ")
```

Comment:
Could put a no input while loop here to catch when a user enters nothing here instead of the program breaking.

Reply:
Noted. I expect I overlooked this. Thank you